bin/zeppelin-daemon.sh

```diff
@@ -94,7 +94,8 @@ function initialize_default_directories() {
     $(mkdir -p "${ZEPPELIN_PID_DIR}")
   fi
 
-  echo "Browse localhost:8080 in your browser"
+  IP_ADDR="$(ifconfig en0 | grep inet | grep -v inet6 | cut -d ' ' -f2)"
+  echo -e "Browse $IP_ADDR:8080 in your browser.\nIf you are testing on your local computer, use localhost:8080"
```

Comment:
@AhyoungRyu Thanks! Which file do I have to refer for this value? `zeppelin-site.xml` or `zeppelin-env.sh` or else?